README.md

```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
 Groups
 ------
 
-When using [ACBS](https://github.com/AOSC-Dev/acbs/) (Autobuild CI Build System),
+When using [ACBS](https://github.com/AOSC-Dev/acbs) (Autobuild CI Build System),
 it is possible to build a series of packages with a single command. This is
 achieved by the "groups" defined in the `groups/` directory, with files defining
 a list of packages to build.
 
-Automation
-----------
+Notes for developers on Windows
+-------------------------------
 
-Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
+Currently, this repository cannot be checked out on Windows, since there're
```

Comment:
Please use "there are," and avoid using abbreviated forms in formal documentation.

Reply:
Resolved.